doc/classes/Shape2D.xml

```diff
@@ -69,7 +69,8 @@
 	</methods>
 	<members>
 		<member name="custom_solver_bias" type="float" setter="set_custom_solver_bias" getter="get_custom_solver_bias" default="0.0">
-			The shape's custom solver bias.
+			The shape's custom solver bias. Defines how much bodies react to enforce contact separation when this shape is involved.
+ 			When set to [code]0.0[/code], the default value of [code]0.3[/code] is used.
```

Comment:
Indentation issue.

Reply:
I don't see the indenation issue – both lines are using tab indentation now.

Reply:
Well the CI is failing, check the log :)
Hint: the line starts with a space.

Reply:
Ah, I see it now. I didn't notice it while selecting text in VS Code (which shows invisible characters when selecting them by default).